extensions/vercast/package.json

```
@@ -2,53 +2,57 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "vercast",
   "title": "Vercel Deployments",
```

I just checked with the team and it is fine to change this to something else, since the extension now does more than deployments. Should we call it "Vercel"?
```suggestion
  "title": "Vercel",
```